File: dotnet/cafdemalihapa/Applications/DirectoryManagement/Commands.cs
namespace cafdemalihapa.Applications
{
    public static class Commands
    {
        public static string Get()
        {
            var command = CommandLineArgs.GetByKey("--command");
            return command;
        }

        public static bool IsCreate(this string command)
        {
            return command == "create";
        }

        public static bool IsOpen(this string command)
        {
            return command == "open";
        }

        public static bool IsShut(this string command)
        {
            return command == "shut";
        }

        public static bool IsDirectoryToBeOpen(this string command)
        {
            return command == "directory-to-be-open";
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Applications/DirectoryManagement/Commands.fs
namespace cafdemwimu.console.Applications

open System.Runtime.CompilerServices
open cafdemwimu.console

[<Extension>]
type Commands =
    static member Get() =
        let command = CommandLineArgs.GetByKey("--command")
        command

    [<Extension>]
    static member IsCreate(command: string) = command = "create"

    [<Extension>]
    static member IsOpen(command: string) = command = "open"

    [<Extension>]
    static member IsShut(command: string) = command = "shut"

    [<Extension>]
    static member IsDirectoryToBeOpen(command: string) = command = "directory-to-be-open"
